### PR TITLE
change styling of the scroll-bar

### PR DESCRIPTION
**Pull request by Roland-Hufnagel**
I changed the styling belonging to the scroll-bar behaviour. Please have a look.

### CODE REVIEW

File: components/Header.js [hunk 1]
```diff
@@ -20,8 +20,7 @@ const StyledHeader = styled.header`
   text-align: center;
   font-family: var(--fontfamily-special);
   font-size: 1.3rem;
-  position: fixed;
```
Comment (kai-franke) [inline components/Header.js:23]: Without `position: fixed` the height of the header increases a bit.

File: components/Header.js [hunk 2]
```diff
@@ -20,8 +20,7 @@ const StyledHeader = styled.header`
...
-  top: 0;
+  height: 100%;
```
Comment (kai-franke) [inline components/Header.js:23]: Do you really need it? I don't see a difference without the `height: 100%`.

Comment (kai-franke) [inline components/Header.js:23]: Do you really need it? I don't see a difference without the `height: 100%`.

File: components/Header.js [hunk 3]
```diff
@@ -20,8 +20,7 @@ const StyledHeader = styled.header`
...
   width: 100%;
   box-shadow: var(--boxshadow-primary);
```
Comment (kai-franke) [inline components/Header.js:26]: Please add a `z-index` because without `position: fixed` the shadow is not shown above the main content.

**Review by kai-franke — APPROVED** [review]
That's a great feature! I love it.
Just some small changes...

File: components/Header.js (hunk 1 above)
Comment (Roland-Hufnagel) [reply components/Header.js:23]: the height of the header is now defined in the AppContainer styling. Ich reduced it to get the same height as before.

File: components/Header.js (hunk 2 above)
Comment (Roland-Hufnagel) [reply components/Header.js]: deleted

File: components/Header.js (hunk 3 above)
Comment (Roland-Hufnagel) [reply components/Header.js:27]: done 👍

**Pull request merged**